src/styles/annotator/adder.scss

```diff
@@ -11,148 +14,136 @@ $adder-transition-duration: 80ms;
   // Adder, even when using Shadow DOM.
   all: initial;
 
-  // Adder entry animation settings
-  animation-duration: $adder-transition-duration;
-  animation-timing-function: ease-in;
-  animation-fill-mode: forwards;
+  @include utils.border;
 
+  position: absolute;
   box-sizing: border-box;
   direction: ltr;
-  position: absolute;
-  background: var.$white;
-  border: 1px solid rgba(0, 0, 0, 0.2);
```

Comment:
The old border had a bit of opacity, but I'm a) not certain why and b) it was so subtle as to be indiscernible to the naked eye.